modules/lightning_telemetry/lightning_telemetry.services.yml

```diff
@@ -0,0 +1,4 @@
+services:
+  lightning_telemetry.telemetry:
+    class: Drupal\lightning_telemetry\Telemetry
+    arguments: ['@extension.list.module', '@http_client', '@config.factory']
```

Comment:
Because we are using `extension.list.module`, this module will NOT work on Drupal 8.5.x or earlier. We should document that, _and_ we should add a hook_requirements() implementation to prevent installation on those old versions.

Reply:
Can't we just add this to lightning_telemetry.info.yml?

```
dependencies:
  - drupal:system (>= 8.6)
```

Reply:
That seems like it should work fine. We should _also_ add a requirement or conflict to `lightning_telemetry/composer.json`.

Reply:
@grasmash we still need a `lightning_telemetry/composer.json` per ^^.